microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,293 @@
+* [CoreWebView2Controller.Close Method](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controller#close)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2::Reload method](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2#reload)
+* [ICoreWebView2Controller::Close method](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2controller#close)
+
+---
+
+If the main frame is not impacted (`FrameRenderProcessExited`), your application can communicate with the main frame to recover content in the impacted frames.  The `ProcessFailed` event provides details for the impacted frames through the `FrameInfosForFailedProcess` property.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ProcessFailedEventArgs.FrameInfosForFailedProcess Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs.frameinfosforfailedprocess)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ProcessFailedEventArgs.FrameInfosForFailedProcess Property](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs#frameinfosforfailedprocess)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ProcessFailedEventArgs2::get_FrameInfosForFailedProcess property](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs2#get_frameinfosforfailedprocess)
+
+---
+
+
+<!-- ------------------------------ -->
+#### The GPU process has exited unexpectedly
+
+The content in your WebView2 controls might flash as the process is automatically recreated. Every WebView2 control in the _WebView2 Process Group_ will receive the `ProcessFailed` event with:
+* **Failure kind:** `GpuProcessExited`.
+* **Failure reason:** any, except `Unresponsive` and `ProfileDeleted`.
+
+This is the most common WebView2 process failure and is auto-recoverable. Your application does **not** need to handle recovery for this event, but can collect information to understand any persistent issues, or if there is an underlying cause for repeated GPU process exits.
+
+
+<!-- ------------------------------ -->
+#### A utility process has exited unexpectedly
+
+There might be some interruptions (for example, if the utility process was hosting the audio service) necessary processes are automatically recreated. Every WebView2 control in the _WebView2 Process Group_ will receive the `ProcessFailed` event with:
+* **Failure kind:** `UtilityProcessExited`.
+* **Failure reason:** any, except `Unresponsive` and `ProfileDeleted`.
+
+This process failure is not fatal and is auto-recoverable. Your application does **not** need to handle recovery for this event, but can collect information to understand any persistent issues, including the `ProcessDescription` provided in the event args.
```

Comment:
We might want to add the reference to `ProcessDescription` like we do for the sections above.

Reply:
locally, added:
```
##### [.NET/C#](#tab/dotnetcsharp)

* [CoreWebView2ProcessFailedEventArgs.ProcessDescription Property](/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs.processdescription)

##### [WinRT/C#](#tab/winrtcsharp)

* [CoreWebView2ProcessFailedEventArgs.ProcessDescription Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs#processdescription)

##### [Win32/C++](#tab/win32cpp)

* [ICoreWebView2ProcessFailedEventArgs2::get_ProcessDescription property](/microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs2#get_processdescription)

---
```
also deleted all LMC url prefixes per doc build